docs/source/concepts/HowToDefineGeometricShape.rst

```diff
@@ -340,6 +452,59 @@ axis runs from the start aperture to the end aperture. "Height" is along
 the y-axis and "width" runs along the x-axis, before the application of
 the "axis" rotation.
 
+Rotating Shapes
+~~~~~~~~~~~~~~~
+
+*Note that some shapes (such as* `Cylinder (finite height) <Cylinder_>`_ *and* `Tapered Guide`_ *) can be oriented in a
+certain direction, using the* ``axis`` *tag, but this does not support all possible 3D rotations.*
+
+Most shapes can be rotated individually or as part of an ensemble, by using the
+``rotate`` or ``rotate-all`` tags respectively.
+The shapes that can be rotated are: Sphere_, `Cylinder (finite height) <Cylinder_>`_,
+`Hollow Cylinder (finite height) <Hollow Cylinder_>`_, `Infinite Cylinder`_,
+`Slice of Cylinder Ring`_, `Infinite Plane`_, Cuboid_, Hexahedron_ and `Tapered Guide`_.
+
+Use the ``rotate`` tag to rotate a shape **individually around its centre** (or centre-of-bottom-base).
+The shape is rotated by an angle in degrees around the x,y and z axes in that order. To rotate a
+cylinder 90° clockwise around x and 45° anti-clockwise around y:
+
+.. code-block:: xml
+
+      <cylinder id="stick">
+        <centre-of-bottom-base x="-0.5" y="0.0" z="0.0" />
+        <axis x="1.0" y="0.0" z="0.0" />
+        <radius val="0.05" />
+        <height val="1.0" />
+        <rotate x="90" y="-45" z="0" />
+      </cylinder>
+
+      <algebra val="stick" />
```

Comment:
Given that the cylinder could be rotated as required using just the axis tag, I think it would be better to use the cuboid here. Please can you update to an example using the width\height\depth representation of a cuboid?